src/bootstrap/test.rs

```diff
@@ -926,7 +926,12 @@ default_test!(Incremental {
     suite: "incremental"
 });
 
-default_test!(Debuginfo { path: "src/test/debuginfo", mode: "debuginfo", suite: "debuginfo" });
+default_test_with_compare_mode!(Debuginfo {
+    path: "src/test/debuginfo",
+    mode: "debuginfo",
+    suite: "debuginfo",
+    compare_mode: "split-dwarf"
```

Comment:
I'm not entirely sure, but does this mean that all the previous debug info tests are now also tested with split dwarf and compared with regular debug info output?

Reply:
There's no stdout or stderr files for debuginfo tests, and so there's no output comparison - it just runs the tests in both modes. It could be that Split DWARF isn't really working and the way we run debuginfo tests hides that, I'm not too sure, but they were failing when I first changed this until I fixed that, so I've got some confidence in it.